hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieDeleteBlock.java

```diff
@@ -76,17 +77,20 @@ public class HoodieDeleteBlock extends HoodieLogBlock {
   private DeleteRecord[] recordsToDelete;
 
   public HoodieDeleteBlock(List<Pair<DeleteRecord, Long>> recordsToDelete,
+                           String baseFileInstantTimeForPositions,
                            boolean shouldWriteRecordPositions,
                            Map<HeaderMetadataType, String> header) {
     this(Option.empty(), null, false, Option.empty(), header, new HashMap<>());
-    if (shouldWriteRecordPositions && !recordsToDelete.isEmpty()) {
```

Comment:
Why not just put the `baseFileInstantTimeForPositions` in the map?

Reply:
Yeah, we can do that to simplify the code here.